test/function_test.py

```diff
@@ -533,7 +561,7 @@ def test_map_exceptions(client, servicer):
 
         res = list(custom_function_modal.map(range(6), return_exceptions=True))
         assert res[:4] == [0, 1, 4, 9] and res[5] == 25
-        assert type(res[4]) is UserCodeException and "bad" in str(res[4])
+        assert type(res[4]) is CustomException and "bad" in str(res[4])
```

Comment:
Breaking change!

Reply:
Although this is breaking, I'll say this is more of a bug fix.